docker/dev.Dockerfile

```diff
@@ -33,10 +33,22 @@ ENV SHELLCHECK_VERSION="0.4.6"
 RUN curl -sSfL "https://storage.googleapis.com/shellcheck/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" \
     | tar -Jx --strip-components=1 -C /usr/local/bin
 
+#Poetry Installation
+ENV POETRY_VERSION=1.0.5
+RUN pip install "poetry==$POETRY_VERSION"
```

Comment:
Why not just `pip install poetry==1.0.5`?

Reply:
We can do that as well, better to pin the version to avoid version mismatch